src/components/sortable-list/index.tsx

```
@@ -0,0 +1,219 @@
+        useSensor( KeyboardSensor, {
+            coordinateGetter : sortableKeyboardCoordinates,
+        })
+    );
+
+    const getSortingStrategy = (): SortingStrategy => {
+        switch ( strategy ) {
+            case 'horizontal':
+                return horizontalListSortingStrategy;
+            case 'grid':
+                return rectSortingStrategy;
+            case 'vertical':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)